pkg/cmd/skip-test/main.go

```diff
@@ -169,6 +182,10 @@ Release note: None
 
 	// The shorthand for opening a web browser with Python, `python -m
 	// webbrowser URL`, does not set the status code appropriately.
+	// TODO(mgartner): A Github username should be used instead of remote in the
+	// URL. A remote might not be the same as the username, in which case an
+	// incorrect URL is opened. For example, the remote might be named "origin"
+	// but the username is "mgartner".
```

Comment:
Haha this is copied straight from backport

Reply:
backport is a bit smarter: https://github.com/benesch/backport/blob/69513f3c0e5b199de33f40c1b230a29d820def14/main.go#L187-L188